bigquery/cloud-client/natality_tutorial.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def run_natality_tutorial():
+    # [START bigquery_query_natality_tutorial]
+    """Create a Google BigQuery linear regression input table.
+
+    In the code below, the following actions are taken:
+    * A new dataset is created "natality_regression."
+    * A new table "regression_input" is created to hold the inputs for our
```

Comment:
We don't have to actually create the table, right? The query job should be able to do that for us, so long as the [`configuration.query.createDisposition` BigQuery job property](https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs) is correct.